docs/docs/cloud/how-tos/stream_updates.md

```diff
@@ -0,0 +1,114 @@
+# How to stream state updates of your graph
+
+LangGraph Cloud supports multiple streaming modes. The main ones are:
+
+- `values`: This streaming mode streams back values of the graph. This is the **full state of the graph** after each node is called.
+- `updates`: This streaming mode streams back updates to the graph. This is the **update to the state of the graph** after each node is called.
+- `messages`: This streaming mode streams back messages - both complete messages (at the end of a node) as well as **tokens** for any messages generated inside a node. This mode is primarily meant for powering chat applications.
+
+
+This guide covers `stream_mode="updates"`.
+
+First let's set up our client and thread:
+
+=== "Python"
+    
+    ```python
+    from langgraph_sdk import get_client
+
+    client = get_client(url="whatever-your-deployment-url-is")
+    # create thread
+    thread = await client.threads.create()
+    print(thread)
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
+    # create thread
```

Comment:
`#` won't work in JS, needs to have `//` for comments (let's make sure to update this everywhere)